### PR TITLE
abella: update to 2.0.8.3; add ocaml ports needed to support it

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Finally, update this one

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: ocaml/ocaml-base64/Portfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           ocaml 1.1
+
+name                ocaml-base64
```
Comment (reneeotten) [inline ocaml/ocaml-base64/Portfile:7]: this is set by the PortGroup already

File: ocaml/ocaml-crunch/Portfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+PortGroup           ocaml 1.1
+
+github.setup        mirage ocaml-crunch 3.3.1 v
+revision            0
+categories          ocaml devel
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+license             ISC
+description         Convert a filesystem into a static OCaml module
+long_description    {*}${description}
+homepage            https://github.com/mirage/ocaml-crunch
```
Comment (reneeotten) [inline ocaml/ocaml-crunch/Portfile:14]: this is set by the PH already

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: ocaml/ocaml-base64/Portfile (hunk 1 above)
Comment (barracuda156) [reply ocaml/ocaml-base64/Portfile:7]: Ok, I will change these today.

**Comment by barracuda156** [comment]
@reneeotten Fixed both issues above now.

**Comment by pmetzger** [comment]
You added three ports, but that's not mentioned in the commit message.

**Comment by barracuda156** [comment]
> You added three ports, but that's not mentioned in the commit message.

@pmetzger Those are required dependencies, not random stuff. But I will change commit message now.

**Comment by pmetzger** [comment]
Not my fave commit message but whatever.

**Comment by barracuda156** [comment]
Thanks for merging!

**Pull request merged**